src/app/analyze/analyze.module.ts

```diff
@@ -7,9 +7,15 @@ import { AnalyzeRoutingModule } from './analyze-routing.module';
 
 import { AnalyzeOverviewModule } from './analyze-overview/analyze-overview.module';
 
+import {RenderStackDetailsComponent} from './stackreports/render-stack-details/render-stack-details.component';
+import {StackModule} from './stackreports/render-stack-details/module';
```

Comment:
My refactor changed the location of the StackDetails module, please can you not move it.